src/translib/common_app.go

```diff
@@ -618,7 +620,11 @@ func checkAndProcessLeafList(existingEntry db.Value, tblRw db.Value, opcode int,
 				if opcode == DELETE {
 					if len(valueLst) > 0 {
 						mergeTblRw.SetList(field, exstLst)
-						delete(tblRw.Field, field)
+						if len(exstLst) == 0 {
+							tblRw.Field[field] = ""
```

Comment:
Same as first comment for common_app. This is already available in dell_sonic branch. Please rebase to dell_sonic before merge.